src/main/java/ai/rev/sentimentanalysis/SentimentAnalysisClient.java

```diff
@@ -0,0 +1,251 @@
+package ai.rev.sentimentanalysis;
+
+import ai.rev.helpers.ClientHelper;
+import ai.rev.speechtotext.ApiClient;
+import ai.rev.speechtotext.models.asynchronous.RevAiJob;
+import ai.rev.speechtotext.models.asynchronous.RevAiJobOptions;
+import ai.rev.sentimentanalysis.models.Sentiment;
+import ai.rev.sentimentanalysis.models.SentimentAnalysisJob;
+import ai.rev.sentimentanalysis.models.SentimentAnalysisJobOptions;
+import ai.rev.sentimentanalysis.models.SentimentAnalysisResult;
+import ai.rev.speechtotext.models.asynchronous.RevAiTranscript;
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The SentimentAnalysisClient object provides methods to send and retrieve information from all the
+ * Rev AI Sentiment Analysis API endpoints using the Retrofit HTTP client.
+ */
+public class SentimentAnalysisClient {
+
+  private OkHttpClient client;
+
+  /** Interface that SentimentAnalysisClient methods use to make requests */
+  public SentimentAnalysisInterface apiInterface;
+  
+  /**
+   * Constructs the API client used to send HTTP requests to Rev AI. The user access token can be
+   * generated on the website at <a
+   * href="https://www.rev.ai/access_token">https://www.rev.ai/access_token</a>.
+   *
+   * @param accessToken Rev AI authorization token associate with the account.
+   * @throws IllegalArgumentException If the access token is null or empty.
+   */
+  public SentimentAnalysisClient(String accessToken) {
+    if (accessToken == null || accessToken.isEmpty()) {
+      throw new IllegalArgumentException("Access token must be provided");
+    }
+    this.client = ClientHelper.createOkHttpClient(accessToken);
+    Retrofit retrofit = ClientHelper.createRetrofitInstance(client, "sentiment_analysis", "v1");
+    this.apiInterface = retrofit.create(SentimentAnalysisInterface.class);
+  }
+  
+  /** Manually closes the connection when the code is running in a JVM */
+  public void closeConnection() {
```

Comment:
Is this necessary?

Reply:
I believe so. @Chanc3 may have some more info on why this is needed